docs/niswitch/functions.rst

```diff
@@ -118,7 +118,7 @@ niswitch.Session methods
 
     :type scan_mode: :py:data:`niswitch.ScanMode`
 
-.. function:: configure_scan_trigger(scan_delay, trigger_input, scan_advanced_output)
+.. function:: configure_scan_trigger(trigger_input, scan_advanced_output, scan_delay)
```

Comment:
opened https://github.com/ni/nimi-python/issues/365 (missing default in documentation)